polygeist/lib/Dialect/Polygeist/Analysis/ReachingDefinitionAnalysis.cpp

```diff
@@ -126,11 +127,8 @@ ChangeResult ReachingDefinition::reset() {
 }
 
 ChangeResult ReachingDefinition::setModifier(Value val, Definition def) {
-  ReachingDefinition::ModifiersTy &mods = valueToModifiers[val];
```

Comment:
The dataflow framework visits operations in affine.for regions more than once.  Had to remove the assert because of that. The assertion was triggered but `test1` in the new `memoryAccess.mlir` file.